ReactiveXComponent/WebSocket/WebSocketMessage.cs

```diff
@@ -16,9 +16,30 @@ public WebSocketMessage(string command, string topic, string json, string compon
             ComponentCode = componentCode;
         }
 
-        public string Command { get; }
-        public string Topic { get; }
-        public string Json { get; }
-        public string ComponentCode { get; }
+        public override bool Equals(object obj)

```

Comment:
Why do you need to implement equality members? Are you using this class as a key in a collection?